### ISSUE TITLE
perl in container causes bus error

### ISSUE THREAD

**Issue by pen** [lkwg82/h2o.docker#8]
## Description

One of my private container is made of h2o-http2-server, adding some perl modules and scripts.
Today I re-built it but doesn't work because perl causes bus error.

Finaly I found it is same to h2o-http2-server itself.

## Revision

https://github.com/lkwg82/h2o.docker/tree/dd8203ce2d6899174d3278e1567bd08776ff2cef
according to https://hub.docker.com/r/lkwg82/h2o-http2-server/builds

## Environment

Macbook Air 13" Mid 2012
Mac OS X 10.14.6
Docker version 19.03.1, build 74b1e89
zsh 5.3

## To Reproduce

```zsh
docker run -p "8080:8080" -ti lkwg82/h2o-http2-server sh
```

on shell in the container

```sh
perl
```

frequency: always

## Note

Removing `compress *.so` part of Dockerfile fixes the problem.

**Comment by pen** [comment]
yet another reproduce:

```
$ docker images -a
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE

$ mkdir tmp
$ cd tmp
$ git clone https://github.com/lkwg82/h2o.docker

(snip)

$ cd h2o.docker/
$ docker build -t h2o-latest .

(snip)

Successfully built c9340d62ae4f
Successfully tagged h2o-latest:latest

$ docker run -ti h2o-latest sh
~ $ perl -v
Bus error
```

back to OS X

```
$ sed -i '/exec upx/d' Dockerfile
$ docker build -t h2o-noupx .

(snip)

Successfully built cce9a693d7f6
Successfully tagged h2o-noupx:latest

$ docker run -ti h2o-noupx sh
~ $ perl -v

This is perl 5, version 28, subversion 2 (v5.28.2) built for x86_64-linux-thread-multi

Copyright 1987-2019, Larry Wall

Perl may be copied only under the terms of either the Artistic License or the
GNU General Public License, which may be found in the Perl 5 source kit.

Complete documentation for Perl, including FAQ lists, should be found on
this system using "man perl" or "perldoc perl".  If you have access to the
Internet, point your browser at http://www.perl.org/, the Perl Home Page.
```

**Comment by lkwg82** [comment]
FYI: I removed automatic builds from docker hub, the images are build on a private build-machine

**Comment by lkwg82** [comment]
Btw. thx for raising this bug

**Comment by lkwg82** [comment]
will only be fixed in v2.2.6

**Comment by lkwg82** [comment]
published on docker hub and tagged as `v2.2.6` and as `latest`: see https://cloud.docker.com/repository/docker/lkwg82/h2o-http2-server/tags

**Comment by pen** [comment]
Thank you!
